app/src/main/java/com/example/encoratask/model/Character.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.encoratask.model
+
+data class Character (
+    var id : Int,
+    var name : String,
+    var status : String,
```

Comment:
Four comments here:
1. Why do you need to map all the values in the JSON to the data class? Do you need them all?
2. In the JSON example status, species and gender are attributes with very specific values (Enums?)
3. I strongly suggest you to set the types as nullables (Int?, String?, etc). You can't assure that API is not null for every attribute in the response